tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>This is okay for now. It's too bad there's not a simpler way for loading data in. If I was seeing this for the first time, I would be put off by the amount of code to load data in, when in fact, it's really simple to build an index from scratch (in terms of lines of code).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I could maybe generate some smaller functions and try calling them all in. One caveat I encountered with milvus lite is that the connection has to be "live" whenever interacting with it</p>